src/figuresClasses.ts

```diff
@@ -1,19 +1,106 @@
 export interface Figure {
-
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
 
-}
+  color: 'red' | 'green' | 'blue';
```

Comment:
As I said above, it's better to create a type for the Color
```suggestion
  color: Color;
```